docs/cow-protocol/tutorials/cow-swap/cancel.mdx

```diff
@@ -1,11 +1,11 @@
 ---
-sidebar_position: 3
+sidebar_position: 5
```

Comment:
nit: i thought after Swap made sense as they are the basic flows, but not strong preference.
Limit orders, twap and native are enhancements